src/02-common-principles.md

```diff
@@ -122,15 +115,28 @@ my_dataset/
     sub-02/
     ...
   derivatives/
+    pipeline_1/
+    pipeline_2/
     ...
 ```
 
-In this example **only the `rawdata` subfolder needs to be BIDS compliant** 
-dataset. This specification does not prescribe anything about the contents of 
-`sourcedata` and `derivatives` folders in the above example - nor does it 
-prescribe the `sourcedata`, `derivatives`, or `rawdata` folder names. The above 
-example is just a convention that can be useful for organizing raw, source, and 
-derived data while maintaining BIDS compliancy of the raw data folder.
+In this example **only `rawdata` subfolder and all subfolders of `derivatives`**
+**need to be BIDS compliant** datasets. This specification does not prescribe 
```

Comment:
So with this BEP the `/derivatives` will be defined for MRI data and it makes sense to then enforce that the contents of `/derivatives` is formatted according to the rules. However,

1. are there any bids-validator functions in place yet?
2. All other modalities will lose the "convenience" of using `/derivatives` as a dump

point 2. is probably fair enough and we'll simply have to work on BEP021.

Reply:
> 1. are there any bids-validator functions in place yet?

Paging @rwblair.

Reply:
Looks like currently any file located in derivatives is accepted:
https://github.com/bids-standard/bids-validator/blob/master/bids-validator/bids_validator/rules/associated_data_rules.json

So there is a function currently checking the directory but, we will need to expand the regexp to account for the standard.